src/facade/dragonfly_connection.cc

```diff
@@ -1212,24 +1214,22 @@ void Connection::HandleMigrateRequest() {
     // We need to return early as the socket is closing and IoLoop will clean up.
     // The reason that this is true is because of the following DCHECK
     DCHECK(!dispatch_fb_.IsJoinable());
+
     // which can never trigger since we Joined on the dispatch_fb_ above and we are
     // atomic in respect to our proactor meaning that no other fiber will
     // launch the DispatchFiber.
     if (!this->Migrate(dest)) {
       return;
     }
   }
-
-  // In case we Yield()ed in Migrate() above, dispatch_fb_ might have been started.
-  LaunchDispatchFiberIfNeeded();
```

Comment:
why did you remove this?

Reply:
we do that inside `OnPostMigrateThread` function.